parameter_files/archive/api37.0.0_011025_patch_params.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<all>
+  <!-- This parameter patch file updates the parameter file with the changes -->
+  <!-- brought in by Charlie Koven related to the new grazing feature -->
+  <!-- Per PR 1140: -->
+
+  <!-- To use this patch file run the following command: -->
+  <!--   Run `python ../tools/UpdateParamAPI.py -/-f archive/api37.0.0_011025_patch_params.xml` -->
+
+  <!-- General xml file usage notes: -->
+  <!-- Special mod types "variable_add": creates a new variable
+		 "dimension_add": creates a new dimension
+		 "variable_del": removes a variable, must include attribute "name" 
+		 "variable_change": change values in the parameter, possibly including the name, and
+                          this must also come with a name attribute as well -->
+  <!-- To change a dimension. The easiest way to script this, is to delete the variables
+		 that use it, delete the dimension. Re-add it with the new size, and then re-add
+		 the variables -->
+  <!-- -->
+  
+  <base_file>archive/api37.0.0_011025_params_default.cdl</base_file>
+  <new_file>fates_params_default.cdl</new_file>
+  <pft_list> 1,2,3,4,5,6,7,8,9,10,11,12,13,14 </pft_list>
+  <mods>
+    <mod type="variable_add">
+      <na> fates_landuse_grazing_palatability </na>
+      <di> fates_pft </di>
+      <un> unitless 0-1 </un>
+      <ln> Relative intensity of leaf grazing/browsing per PFT </ln>
+      <val> 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1 </val>
+    </mod>
+    <mod type="variable_add">
+      <na> fates_landuse_grazing_rate </na>
+      <di> fates_landuseclass </di>
+      <un> 1/day </un>
+      <ln> fraction of leaf biomass consumed by grazers per day </ln>
+      <val> 0.0, 0.0, 0.0, 0.0, 0.0 </val>
+    </mod>
```

Comment:
@ckoven we should add a fates grazing test that checks non-zero grazing rates at some point in the near future.  Can you recommend some  parameter values for the testing?